style/breakpoints.css

```diff
@@ -13,3 +13,7 @@
   top: -1px;
   color: #f22;
 }
+
+.highlight {
+  background-color: orange;
```

Comment:
Here we could make use of a JupyterLab CSS variable (to also handle the dark theme)